services/director-v2/src/simcore_service_director_v2/models/comp_runs.py

```diff
@@ -62,6 +62,13 @@ def ensure_utc(cls, v: datetime.datetime | None) -> datetime.datetime | None:
             v = v.replace(tzinfo=datetime.timezone.utc)
         return v
 
+    @validator("metadata", pre=True)
+    @classmethod
+    def convert_null_to_empty_metadata(cls, v):
```

Comment:
THOUGHT: in the API i had to pre-process many inputs because of the loose way we employ `None`. At some point, we should review the details to avoid all this unnecessary overhead . It is not only very error prone but also consumes resources.

FYI: It happened so many times that I collected the most common `pre-validators` in `https://github.com/ITISFoundation/osparc-simcore/blob/f2f6a81fa9c4021d44c4d77a2fd196a956e6e54b/packages/models-library/src/models_library/utils/common_validators.py

Reply:
well it is here a clear backwards compatibility issue.